packages/model/src/-private/model.ts

```diff
@@ -1740,7 +1740,7 @@ class Model extends EmberObject implements MinimalLegacyRecord {
     this.eachComputedProperty((name, meta) => {
       if (isAttributeSchema(meta)) {
         assert(
-          "You may not set `id` as an attribute on your model. Please remove any lines that look like: `id: attr('<type>')` from " +
+          "You may not set 'id' as an attribute on your model. Please remove any lines that look like: `id: attr('<type>')` from " +
```

Comment:
Making this error message match the other one so the tests don't care about where specifically the error is thrown.